model_analyzer/model_metadata.py

```diff
@@ -140,7 +140,7 @@ def analyze_super_resolution_inputs(self) -> Dict[str, str]:
             return roles
 
         dims = {}
-        for candidate in self.input_layers:
```

Comment:
Can you check if we still need `input_layers`

Reply:
I didn't catch you, could you please explain in detail